service/integration/kas_registry_test.go

```diff
@@ -14,13 +14,13 @@ import (
 	"github.com/stretchr/testify/suite"
 )
 
-var nonExistentKasRegistryId = "78909865-8888-9999-9999-000000654321"
+var nonExistentKasRegistryID = "78909865-8888-9999-9999-000000654321"
 
 type KasRegistrySuite struct {
 	suite.Suite
 	f   fixtures.Fixtures
 	db  fixtures.DBInterface
-	ctx context.Context
+	ctx context.Context //nolint:containedctx // context is used in the test suite
```

Comment:
Should we just go lite on the linting for tests and integrations?

Reply:
We can exclude linters with golangci on `_test.go` files. But probably should try  to fix as much as we can in tests for consistency.

https://golangci-lint.run/usage/configuration/#issues-configuration